experimental/dds/tree2/src/feature-libraries/schema-aware/schemaAware.ts

```diff
@@ -146,7 +146,7 @@ export type TypedFields<
 > = [
 	TFields extends { [key: string]: FieldSchema }
 		? {
-				[key in keyof TFields]: TypedField<
+				-readonly [key in keyof TFields]: TypedField<
```

Comment:
What is the purpose of this?

Reply:
It makes the bubble bench demo still compile. Without it, its properties lose their mutability. Changes in struct schema inference make the struct schema tend to be readonly now, and this type used to blindly preserve that when generating the API.

Reply:
I see, thanks!